apps/common/models/auth-token.js

```diff
@@ -25,13 +26,8 @@ module.exports = class AuthToken extends Model {
       method: 'POST'
     };
 
-    return new Promise((resolve, reject) => {
-      this._request(tokenReq, (err, response) => {
-        if (err) {
-          return reject(err);
-        }
-        return resolve({ bearer: JSON.parse(response.body).access_token });
-      });
+    return this._request(tokenReq).then(response => {
+      return { bearer: response.data.access_token };
```

Comment:
this is much better

out of scope but I think I know how to improve this with async await but if we keep doing that we will never complete the axios work.  I'll tell you what, create a ticket for me to improve all this legacy code with async await

Reply:
I have updated this model to async await  now but will leave the other models for the  improvements ticket